frontend/src/modules/SaleProducts/SaleProducts.tsx

```diff
@@ -32,6 +32,7 @@ const SaleProducts: React.FC = () => {
           size={"large"}
           backgroundImage={shirt}
           priceBeforeSale={45}
+          isPreview={false}
```

Comment:
Consider making isPreview an optional field and setting its default value to false in the ProductCard component.